src/applications/edu-benefits/1995-STEM/config/form.js

```diff
@@ -0,0 +1,251 @@
+import _ from 'lodash/fp';
+
+import fullSchema1995Stem from 'vets-json-schema/dist/22-1995-STEM-schema.json';
+
+import { transform } from '../helpers';
+
+import { urlMigration } from '../../config/migrations';
+
+import * as address from '../../../../platform/forms/definitions/address';
+import FormFooter from '../../../../platform/forms/components/FormFooter';
+import environment from '../../../../platform/utilities/environment';
+import GetFormHelp from '../../components/GetFormHelp';
+import ErrorText from '../../components/ErrorText';
+import preSubmitInfo from '../../../../platform/forms/preSubmitInfo';
+
+import educationTypeUISchema from '../../definitions/educationType';
+import serviceBefore1977UI from '../../definitions/serviceBefore1977';
+import * as toursOfDuty from '../../definitions/toursOfDuty.jsx';
+
+import createContactInformationPage from '../../pages/contactInformation';
+import createOldSchoolPage from '../../pages/oldSchool';
+import createDirectDepositChangePage from '../../pages/directDepositChange';
+import createApplicantInformationPage from '../../../../platform/forms/pages/applicantInformation';
+
+import { showSchoolAddress } from '../../utils/helpers';
+import { benefitsLabels } from '../../utils/labels';
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+const {
+  benefit,
+  civilianBenefitsAssistance,
+  educationObjective,
+  nonVaAssistance,
+} = fullSchema1995Stem.properties;
+
+const {
+  preferredContactMethod,
+  educationType,
+  date,
+  dateRange,
+  serviceBefore1977,
+} = fullSchema1995Stem.definitions;
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: `${environment.API_URL}/v0/education_benefits_claims/1995`,
```

Comment:
Does this need to change?

Reply:
It will, the vets-api changes are still in progress and haven't been merged in and since this form is currently a duplicate of 1995 with a duplicate schema the submission will go through